tgui/packages/tgui/interfaces/ChemHeater.js

```diff
@@ -133,33 +131,33 @@ export const ChemHeater = (props, context) => {
                       format={value => toFixed(value) + ' K'} />
                   ) || '—'}
                 </Box>
-              </TableCell>
-              <TableCell collapsing color="label">
+              </Table.Cell>
+              <Table.Cell collapsing color="label">
                 Basic:
-              </TableCell>
-              <TableCell>
+              </Table.Cell>
+              <Table.Cell>
                 <Button
                   icon={'syringe'}
                   tooltip={'Inject'}
-                  tooltipPosition={"left"}                  
+                  tooltipPosition={"left"}
                   disabled={!basicBufferVol}
                   onClick={() => act('basicBuffer', {
                     target: 1,
                   })} />
-              </TableCell>
-              <TableCell color={"#3853a4"} textAlign="center">
-                {basicBufferVol+"u"}
-              </TableCell>
-              <TableCell>
+              </Table.Cell>
+              <Table.Cell color={"#3853a4"} textAlign="center">
```

Comment:
Would be nice if `#3853a4` was a const that tells what kind of color that is.

Reply:
Done!